### PR TITLE
Fix configuration info

### PR DESCRIPTION
**Pull request by pyrech**

### CODE REVIEW

File: src/DependencyInjection/Providers/GoogleProviderConfigurator.php [hunk 1]
```diff
@@ -19,10 +19,10 @@ public function buildConfiguration(NodeBuilder $node)
         // todo - add the comments as help text, and render in README
         $node
             ->scalarNode('access_type')
-                ->info('Optional value for sending hd parameter. More detail: https://developers.google.com/accounts/docs/OAuth2Login#hd-param')
+                ->info('Optional value for sending access_type parameter. More detail: https://developers.google.com/identity/protocols/OAuth2WebServer#offline')
```
Comment (bocharsky-bw) [inline src/DependencyInjection/Providers/GoogleProviderConfigurator.php]: What about a link to the auth params instead?
https://developers.google.com/identity/protocols/OpenIDConnect#authenticationuriparameters

Comment (pyrech) [inline src/DependencyInjection/Providers/GoogleProviderConfigurator.php]: Updated both url (sigh there is no anchor on access_type row :cry:)

Comment (bocharsky-bw) [inline src/DependencyInjection/Providers/GoogleProviderConfigurator.php]: Yeah, I wish it was :/

**Comment by bocharsky-bw** [comment]
@pyrech Thank you for these tweaks!

**Comment by weaverryan** [comment]
Awesome 👍

**Pull request merged**